internal/service/ecs/task_definition.go

```diff
@@ -613,6 +645,20 @@ func flattenPlacementConstraints(pcs []*ecs.TaskDefinitionPlacementConstraint) [
 	return results
 }
 
+func flattenRuntimePlatform(pc *ecs.RuntimePlatform) []map[string]interface{} {
+	if pc == nil {
+		return nil
+	}
+
+	config := make(map[string]interface{})
+	config["operating_system_family"] = aws.StringValue(pc.OperatingSystemFamily)
```

Comment:
same as expand

Reply:
I've done expand; which ironically worked empty anyway (going off my local test harness); the read-side though seems fine with emptys as things like ipc_mode are optional and theres no checks around these string values / StringValue protects against null 

```
func StringValue(v *string) string {
	if v != nil {
		return *v
	}
	return ""
}
```